integration-tests/utils/integration-helpers.js

```diff
@@ -27,38 +27,56 @@ const clipToken = str => str.toString().substr(-10, 10)
 const testWorkspaceNamePrefix = 'terra-ui-test-workspace-'
 const getTestWorkspaceName = () => `${testWorkspaceNamePrefix}${uuid.v4()}`
 
+
 const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
   const workspaceName = getTestWorkspaceName()
   try {
-    await page.evaluate((name, billingProject) => {
-      return window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+    const response = await page.evaluate((name, billingProject) => {
+      return new Promise(async (resolve, reject) => {
+        return await window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+          .then(resp => resolve(resp))
+          .catch(async err => reject(await err.text()))
+      })
     }, workspaceName, billingProject)
 
     console.info(`Created workspace: ${workspaceName}`)
+    console.info(response)
   } catch (e) {
-    throw Error(`Failed to create workspace: ${workspaceName} with billing project ${billingProject}`)
+    console.error(`Failed to create workspace: ${workspaceName} with billing project: ${billingProject}`)
+    console.error(e)
+    throw e
   }
   return workspaceName
 })
 
+
 const deleteWorkspace = withSignedInPage(async ({ page, billingProject, workspaceName }) => {
   try {
-    await page.evaluate((name, billingProject) => {
-      return window.Ajax().Workspaces.workspace(billingProject, name).delete()
+    const response = await page.evaluate((name, billingProject) => {
+      return new Promise(async (resolve, reject) => {
+        return await window.Ajax().Workspaces.workspace(billingProject, name).delete()
+          .then(async resp => resolve(await resp.text()))
+          .catch(err => reject(err))
+      })
```

Comment:
This looks like it could be simplified

```suggestion
    const response = await page.evaluate(async (name, billingProject) => {
      const response = await window.Ajax().Workspaces.workspace(billingProject, name).delete()
      return response.text()
```

Reply:
Ran some testing with minor change to the suggested code. Everything works! Thank you.